tracer/test/snapshots/OracleTests.SchemaV0.verified.txt

```diff
@@ -0,0 +1,1871 @@
+﻿[
```

Comment:
Have you checked these look as expected? e.g. you can view [the snapshot viewer here](https://andrewlock.github.io/DatadogSpanSnapshotViewer/?q=https%3A%2F%2Fgithub.com%2FDataDog%2Fdd-trace-dotnet%2Fblob%2F3f9b5e9b5c8f5868721e51ccf78c0a3ec0a54b7c%2Ftracer%2Ftest%2Fsnapshots%2FOracleTests.SchemaV0.verified.txt) Unfortunately, because you don't include _all_ spans in the snapshot, you lose the proper relationships between the spans

Reply:
do you think it'd be better to keep all spans in the snapshot to see the links between traces ?

Reply:
ok it took me a while to understand the Perfetto UI (I didn't realize I needed to pan/expand the window from the timeline at the top), but it looks good now !